tests/ui/parser/nested-block-comments.rs

```diff
@@ -0,0 +1,36 @@
+//! Test that nested block comments are properly supported by the parser.
+//!
+//! This is a historical test from the early days of Rust development when
+//! multi-line comment support was first implemented.
+//!
```

Comment:
```suggestion
```

"Historical test" makes it sound like this isn't relevant anymore, but it definitely is. Even tests added a long time ago still help us catch bugs.